content/applications/finance/accounting/others/analytic.rst

```diff
@@ -7,6 +7,4 @@ Analytic
 .. toctree::
    :titlesonly:
 
-   analytic/usage
-   analytic/timesheets
-   analytic/purchases_expenses
+   analytic/analytic_accounting
```

Comment:
Couldn't these two be merged now?
![image](https://user-images.githubusercontent.com/15896813/223455925-f1d26f5b-b3b2-422d-8301-feb98e4316e3.png)